src/plugins/formvalid/formvalid-server-en.hbs

```diff
@@ -121,7 +107,35 @@
 						<legend class="required"><span class="field-name">Signature</span> <strong class="required">(Required field)</strong></legend>
 						<div class="checkbox"><label for="MainContent_wbCheckBoxList1_checkBoxList1_0"><input name="checkBoxList1" class="required_js valid" id="MainContent_wbCheckBoxList1_checkBoxList1_0" required="required" type="checkbox" checked="checked" value="checkBoxList1_0v"><span> <span class="field-name">Print this page</span></span> </label></div>
 						<div class="checkbox"><label for="MainContent_wbCheckBoxList1_checkBoxList1_1"><input name="checkBoxList1" id="MainContent_wbCheckBoxList1_checkBoxList1_1" type="checkbox" value="checkBoxList1_1v"><span> <span class="field-name">Please send me more information</span></span> </label></div>
-						<div class="checkbox required"><label for="MainContent_wbCheckBoxList1_checkBoxList1_2"><input name="checkBoxList1" id="MainContent_wbCheckBoxList1_checkBoxList1_2" aria-invalid="false" required="required" type="checkbox" value="checkBoxList1_2v"><span> <span class="field-name">I agree with the directives</span><strong class="required"> (Required field)</strong></span><span class="label label-danger wb-server-error" id="MainContent_wbCheckBoxList1_checkBoxList1_2Cv"><strong><span class="prefix">Error&nbsp;4: </span>This field is mandatory</strong></span></label></div>
+						<div class="checkbox required"><label for="MainContent_wbCheckBoxList1_checkBoxList1_2"><input name="checkBoxList1" id="MainContent_wbCheckBoxList1_checkBoxList1_2" aria-invalid="false" required="required" type="checkbox" value="checkBoxList1_2v"><span> <span class="field-name">I agree with the directives</span><strong class="required"> (Required field)</strong></span><span class="label label-danger wb-server-error" id="MainContent_wbCheckBoxList1_checkBoxList1_2Cv">This field is mandatory</span></label></div>
+					</fieldset>
+
+					<fieldset class="form-inline chkbxrdio-grp">
+						<legend class="required"><span class="field-name">Work Status</span> <strong class="required">(Required field)</strong><span class="label label-danger wb-server-error" id="workError">You must select No for this question.</span></legend>
+						<label class="radio-inline">
+							<input class="input-validation-error" data-msg-required="You must select Yes or No." data-rule-required="true" id="workNo" name="workStatus" type="radio" value="No" />
+							No
+						</label>
+						<label class="radio-inline">
+							<input class="input-validation-error" id="workYes" checked="checked" name="workStatus" type="radio" value="Yes" />
+							Yes
+						</label>
+					</fieldset>
+
```

Comment:
Please add another example for `.checkbox-inline`.

```suggestion
<fieldset class="form-inline chkbxrdio-grp">
	<legend class="required"><span class="field-name">Owned pets</span> <strong class="required">(Required field)</strong><span class="label label-danger wb-server-error" id="petError">A deer is not a pet.</span></legend>
	<label class="checkbox-inline"><input type="checkbox" checked="checked" name="petChoice" id="petDeer" required="required" /> Deer</label>
	<label class="checkbox-inline"><input type="checkbox" name="petChoice" id="petCat" /> Cat</label>
	<label class="checkbox-inline"><input type="checkbox" name="petChoice" id="petDog" /> Dog</label>
</fieldset>
```

Reply:
example added